usage/usage.go

```diff
@@ -23,20 +24,29 @@ type orgstat struct {
 // tracks for every org
 type Usage struct {
 	sync.Mutex
-	period uint32
-	now    map[int]orgstat
-	prev   map[int]orgstat
-	stop   chan struct{}
+	period    uint32
+	now       map[int]orgstat
+	prev      map[int]orgstat
+	stop      chan struct{}
+	shardID   string
+	partition int32
 }
 
-func New(period uint32, m mdata.Metrics, i idx.MetricIndex, cl clock.Clock) *Usage {
+func New(period uint32, m mdata.Metrics, i idx.MetricIndex, cl clock.Clock, partitions []int32) *Usage {
 	metrics = m
 	metricIndex = i
 	Clock = cl
+	partition := partitions[0]
```

Comment:
That's an edge case, but if i see that right a misconfigured `ClusterManager` could return a slice of `0` elements as partitions.